.github/workflows/rm-closed-pr-images.yaml

```diff
@@ -0,0 +1,28 @@
+name: Remove docker images for stale/closed PR(s).
+on:
+    pull_request:
+        branches:
+            - main
+        types: [closed]
+jobs:
+    cleanup:
+        name: PR - cleanup pr-${{ github.event.number }} images
+        runs-on: ubuntu-latest
+        permissions:
+            packages: write
```

Comment:
I don't know if this will be enough.
In the past you need a PAT to delete images.

Reply:
let's try, I'll send a `[DELETE]` commit with `push` trigger to test removing an already closed `pr-NN`

Reply:
https://github.com/sunodo/sunodo/actions/runs/5094268246/jobs/9157866456#step:2:47

worked

```
  Dry-run pruning of: {"id":96281491,"name":"sha256:e772d593559a6128c0ced77a5776d00afd8576c72e00006a51c9d45c27580b65","created_at":"2023-05-25T17:32:53Z","tags":["pr-54"]}
```